simplekv/cache.py

```diff
@@ -110,6 +110,26 @@ def open(self, key):
             # cache error, ignore completely and return from backend
             return self._dstore.open(key)
 
+    def _copy(self, source, dest):
+        """Implementation of :meth:`~simplekv.CopyMoveMixin.copy`.
+        
+        Copies the data in the backing store and removes the destination key from the cache,
+         in case it was already populated.
+         """
+        k = self._dstore._copy(source, dest)
+        self.cache.delete(dest)
```

Comment:
My comment is superseded by the discussion above about using copy/move if possible and then falling back to using the cache instead of hitting the backend directly.